e2e/install/cli/install_test.go

```diff
@@ -55,6 +55,16 @@ func TestBasicInstallation(t *testing.T) {
 		Eventually(PlatformConditionStatus(ns, v1.IntegrationPlatformConditionReady), TestTimeoutShort).
 			Should(Equal(corev1.ConditionTrue))
 
+			// Check if default security context has been applyed
+		Eventually(OperatorPodHas(ns, func(pod *corev1.Pod) bool {
```

Comment:
I wonder if these checks should be extended to OLM and Helm installation tests as well.

Reply:
That's the target, I wanted to check the test was actually correct before extending it. But as it is I think I have some issues to fix first.

Reply:
Okey. Just let me know when it's good to merge then. Thanks!